src/test/java/edu/cornell/kfs/module/receiptProcessing/service/CuReceiptProcessingServiceImplNegativeTest.java

```diff
@@ -28,66 +26,66 @@
         receiptProcessingService.setBatchInputFileService(batchInputFileService);
+        receiptProcessingService.setBatchInputFileTypes(setupBatchInputFileTypes());
+
+        createDirectory(BATCH_DIRECTORY);
+        copyFile(DATA_FILE_PATH, BATCH_DIRECTORY + "/receiptProcessing_test.csv");
+        createDoneFile();
+        createDirectory(RECEIPT_DIR);
+        copyFile(IMG_FILE_PATH, RECEIPT_DIR + "/testUnit.pdf");
+    }
 
+    private List<BatchInputFileType> setupBatchInputFileTypes() {
         ReceiptProcessingCSVInputFileType batchInputFileType = new ReceiptProcessingCSVInputFileType();
         batchInputFileType.setDirectoryPath(BATCH_DIRECTORY);
         batchInputFileType.setFileExtension("csv");
         batchInputFileType.setCsvEnumClass(ReceiptProcessingCSV.class);
         List<BatchInputFileType> batchInputFileTypeList = new ArrayList<>();
         batchInputFileTypeList.add(batchInputFileType);
 
-        receiptProcessingService.setBatchInputFileTypes(batchInputFileTypeList);
+        return batchInputFileTypeList;
+    }
 
-        //make sure we have a batch directory
-        File batchDirectoryFile = new File(BATCH_DIRECTORY);
+    private void createDirectory(String batchDirectory) {
+        File batchDirectoryFile = new File(batchDirectory);
         batchDirectoryFile.mkdir();
+    }
 
-        //copy the data file into place
-        File dataFileSrc = new File(DATA_FILE_PATH);
-        File dataFileDest = new File(BATCH_DIRECTORY + "/receiptProcessing_test.csv");
+    private void copyFile(String dataFilePath, String pathname) throws IOException {
+        File dataFileSrc = new File(dataFilePath);
+        File dataFileDest = new File(pathname);
         FileUtils.copyFile(dataFileSrc, dataFileDest);
+    }
 
-        //create .done file
+    private void createDoneFile() throws IOException {
         String doneFileName = BATCH_DIRECTORY + "/receiptProcessing_test.done";
         File doneFile = new File(doneFileName);
         if (!doneFile.exists()) {
             LOG.info("Creating done file: " + doneFile.getAbsolutePath());
             doneFile.createNewFile();
         }
-        
-        //make sure we have a pdf directory
-        File pdfDirectoryFile = new File(RECEIPT_DIR);
-        pdfDirectoryFile.mkdir();
+    }
 
-        //copy the data file into place
-        File imgFileSrc = new File(IMG_FILE_PATH);
-        File imgFileDest = new File(RECEIPT_DIR + "/testUnit.pdf");
-        FileUtils.copyFile(imgFileSrc, imgFileDest);
+    @After
+    public void tearDown() throws Exception {
+        deleteDirectory(BATCH_DIRECTORY);
+        deleteDirectory(RECEIPT_DIR);
     }
 
-    @Override
-    protected void tearDown() throws Exception {
-        File batchDirectoryFile = new File(BATCH_DIRECTORY);
+    private void deleteDirectory(String batchDirectory) {
+        File batchDirectoryFile = new File(batchDirectory);
         batchDirectoryFile.delete();
```

Comment:
Yeah, let me look into that - I think we may be doing that in another test class.